public/app/percona/dbaas/components/DBCluster/DBClusterLogsModal/DBClusterLogsModal.test.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import { dataQa } from '@percona/platform-core';
+import { DBClusterLogsModal } from './DBClusterLogsModal';
+import { dbClustersStub } from '../__mocks__/dbClustersStubs';
+import { DBClusterService } from '../__mocks__/DBCluster.service';
+
+jest.mock('../DBCluster.service');
+
+describe('DBClusterLogsModal::', () => {
+  it('should render logs', async () => {
+    let root: ReactWrapper;
+
+    await act(async () => {
+      root = mount(<DBClusterLogsModal isVisible setVisible={jest.fn()} dbCluster={dbClustersStub[0]} />);
+    });
+
+    root.update();
+    expect(root.find(dataQa('dbcluster-pod-logs')).length).toBeGreaterThan(0);
+  });
+  it('should expand logs', async () => {
+    let root: ReactWrapper;
+
+    await act(async () => {
+      root = mount(<DBClusterLogsModal isVisible setVisible={jest.fn()} dbCluster={dbClustersStub[0]} />);
+    });
+
+    root.update();
+
+    expect(root).not.toContain('pre');
+
+    const expandButton = root
+      .find(dataQa('dbcluster-logs-actions'))
+      .find('button')
+      .at(0);
+
+    await act(async () => {
+      expandButton.simulate('click');
+    });
+
+    root.update();
+
+    expect(root.find(dataQa('dbcluster-logs')).length).toBeGreaterThan(0);
```

Comment:
Just to make sure the flow is completely tested, could we assert the logs are collapsed before the click?